src/core/schema.py

```diff
@@ -0,0 +1,11 @@
+import graphene
```

Comment:
this module (`schema`) seems to be GraphQL-specific but its naming doesn't reflect that, so it could be confused with the relational schema

Reply:
I was following the naming convention used in [graphene-python](http://graphene-python.org/), but yes I agree, its very broad and could be named differently.

Reply:
I had the same thoughts, lets change it.